guides/common/modules/proc_using-ansible-job-templates-for-package-and-errata-actions.adoc

```diff
@@ -0,0 +1,17 @@
+[id="Using_Ansible_Job_Templates_for_Package_and_Errata_Actions{context}"]
+= Using Ansible job templates for package and errata actions
```

Comment:
Perhaps something like "Switching provider for REX jobs"

I'm not sure the actions are limited to packages and errata.
Can you try to find an umbrella term for the actions that you can change the provider for?

Reply:
cc @adamruzicka, make sure we use the "provider" term correctly if we go with that :)

Reply:
> something like "Switching provider for REX jobs"

It sounds a bit overly general. We have job templates and each job template is bound to a single provider. When you run a job you just select a single template, you don't really fiddle with providers.

Remote execution provides `Remote Execution Features` which are just shortcuts tying together a thing a user might want to do with a particular job template. What is being described here is how to attach a different job template to a particular rex feature. We also have much fewer rex features than job templates and I'm not sure if we can come up with a definition that would cover all rex features we have.

I would be fine with the current wording. If you really want to squeeze provider in there, then something like "Using ansible provider for package and errata actions" would work for me.